packages/js/email-editor/src/components/header/test/send-button.spec.tsx

```diff
@@ -16,28 +20,6 @@ import { SendButton } from '../send-button';
 import { storeName } from '../../../store';
 import { recordEvent } from '../../../events';
 
-jest.mock( '@wordpress/data', () => ( {
-	useSelect: jest.fn(),
-	createRegistrySelector: jest.fn(),
-} ) );
-
-jest.mock( '@wordpress/core-data', () => ( {
-	createSelector: jest.fn(),
-} ) );
-
-jest.mock( '@wordpress/preferences', () => ( {
-	combineReducers: jest.fn(),
-} ) );
-
-jest.mock( '@wordpress/editor', () => ( {
-	useEntitiesSavedStatesIsDirty: jest.fn(),
-} ) );
-
-jest.mock( '@wordpress/blocks', () => ( {
-	serialize: jest.fn(),
-	parse: jest.fn(),
-} ) );
-
 jest.mock( '@wordpress/hooks', () => ( {
```

Comment:
How we decide what goes to shared mocks? 
```
jest.mock( '@wordpress/hooks', () => ( {	jest.mock( '@wordpress/hooks', () => ( {
	applyFilters: jest.fn( ( _, value ) => value ),		applyFilters: jest.fn( ( _, value ) => value ),
} ) );
```
⬆️ was not added to the shared mocks. I'm trying to figure out, which mocks should go there.

Reply:
In my opinion, each mock that we would repeat multiple times and that does not affect other tests could be there.
It was just an idea to move those mocks into one place. One candidate that should probably not be there is the `@wordpress/components` package because those mocks can be specific for each test.

I suppose `@wordpress/hooks` can be included as well.